### PR TITLE
aws install for server 4.0

### PR DESCRIPTION
**Pull request by lloydChris**
# Description
Corrections for a 4.0 install:

- add instructions for GHE default token
- Correct Certbot command
- add instructions for IRSA

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc [hunk 1]
```diff
@@ -247,7 +252,7 @@ ifndef::env-gcp[]
 . If you are using AWS Route 53 for DNS, you need the *certbot-route53* plugin installed. You can install the plugin with the following command:
 +
 ```shell
-pip3 install certbot-dns-route53
+pip install certbot-dns-route53
```
Comment (lloydChris) [inline jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc:255]: I am not sure this is right, but i could not get `pip3` to work.  Please help me if you know the answer

Comment (lloydChris) [reply jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc]: that works, thanks

**Review by ryanwohara — APPROVED** [review]

**Review by jbialy — APPROVED** [review]
👍

File: jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc [hunk 2]
```diff
@@ -235,7 +235,12 @@ image::github-oauth-new.png[Screenshot showing setting up a new OAuth app]
 .Client ID and Secret
 image::github-clientid.png[Screenshot showing GitHub Client ID]
 
-NOTE: If using GitHub Enterprise, you also need a personal access token and the domain name of your GitHub Enterprise instance.
+=== GitHub Enterprise
+
+If using GitHub Enterprise, you also need a personal access token and the domain name of your GitHub Enterprise instance.
+
+Create the `defaultToken` by navigating to **Settings > Developer Settings > Personal access tokens**. The default token requires no scopes.  You need this value when you configure CircleCI server.
```
[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc:263]: @lloydChris ?

**Pull request merged**